db/views/keyword_stats_v01.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+  keyword_id,
+  COUNT(*) FILTER (WHERE kind = 'ads' AND position = 'top') as top_ads_count,
+  COUNT(*) FILTER (WHERE kind = 'ads') as total_ads_count,
+  COUNT(*) FILTER (WHERE kind = 'non_ads') as non_ads_count
```

Comment:
While scenic is the right choice to implement DB views, the current use case is relatively too simple to bring in such a gun 😅 Showing the total can also be achieved by a query object (see https://github.com/nimblehq/mia-web/tree/752f45a860b25bdc456c1bcc82b6b5a6ca18e334/engines/mia_admin/app/queries/mia_admin). 

Prefer to use DB views for complex reporting features 💡

Reply:
Got it. The reason I go with view in the first place is that it reduces the amount of database queries. However, factoring in the added complexity, it doesn't seem to worth the trade off.

I will go with query objects approach